### ISSUE TITLE
g4-tankresponse - broken examples (Trac #1314)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1308]
both of the example scripts rely on `pmt-simulator` and `DOMSimulator` which have been dropped from simulation

Migrated from https://code.icecube.wisc.edu/ticket/1314
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:03", 
    "description": "both of the example scripts rely on `pmt-simulator` and `DOMSimulator` which have been dropped from simulation", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335643235016", 
    "component": "combo simulation", 
    "summary": "g4-tankresponse - broken examples", 
    "priority": "blocker", 
    "keywords": "examples", 
    "time": "2015-08-30T03:32:06", 
    "milestone": "", 
    "owner": "jgonzalez", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150830T15:01:07: 
* olivas changed milestone from "" to "IceSim 4.2"
* olivas changed priority from "normal" to "blocker"
* olivas commented:

> Broken example scripts are blockers.

**Comment by icecube-trac** [comment]
Trac update at 20150916T17:13:48: 
* hdembinski commented:

> In changeset 137630:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137630"
> removed outdated and broken examples, which duplicated the icetop simulation segment from topsimulator, closes #1308
> ```


* hdembinski changed resolution from "" to "fixed"
* hdembinski changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:03: 
* olivas commented:

> Milestone IceSim 4.2 deleted


* olivas changed milestone from "IceSim 4.2" to ""
